### ISSUE TITLE
Does `add_embedding` support word embedding with metadata that has an header?

### ISSUE THREAD

**Issue by MarcCote** [lanpa/tensorboard-pytorch#68]
When I try to use `writer.add_embedding` for my word embeddings the following assertion is raised: ["#labels should equal with #data points"](https://github.com/lanpa/tensorboard-pytorch/blob/master/tensorboardX/writer.py#L445). However, according to [tensorboard documentation](https://www.tensorflow.org/programmers_guide/embedding#metadata) the metadata can have one additional row: the header.

As a test, I've commented out line 445 and it works without any issue. I was able to visualize my word embeddings in Tensorboard.

**Comment by lanpa** [comment]
Hi, my code is based on the assumption in tensorboard doc:
` If the TSV metadata file has only a single column, then we don’t expect a header row, and assume each row is the label of the embedding. We include this exception because it matches the commonly-used "vocab file" format.`

Maybe your header message is hiding in your visualization lol.

**Comment by lanpa** [comment]
implemented in #110
